memoize/wrapper.py

```diff
@@ -115,15 +115,15 @@ async def refresh(actual_entry: Optional[CacheEntry], key: CacheKey,
                         asyncio.ensure_future,
                         try_release(to_release, configuration_snapshot)
                     )
-            except Exception as e:
+            except BaseException as e:
```

Comment:
This is another case where we're calling only internal code, so any nonstandard exception would be something coming from the runtime that we aren't expected to handle.